rita/src/rita_client/traffic_watcher/mod.rs

```diff
@@ -81,8 +81,8 @@ pub fn watch<T: Read + Write>(
     let (input, output) = KI.read_iface_counters("wg_exit")?;
 
     // account for wg packet overhead
-    let input = input.0 + input.1 * 80;
-    let output = output.0 + output.1 * 80;
+    let input = input.total_bytes();
```

Comment:
as a fundamental question do we really want to account for overhead? Does that lend itself to better pricing or does it just cancel out?

Reply:
I'd say yes because thats what we actually transmit, but we see here only 'raw' traffic unpacked from tunnel.